.github/workflows/configure.sh

```diff
@@ -45,7 +45,7 @@ elif [ "${maker}" = "cmake" ]; then
     # Intel icpx needs -Wno-unused-command-line-argument to avoid
     # warnings: 'linker' input unused, which prevent CMake finding OpenMP.
     cmake -Dcolor=no \
-          -DCMAKE_CXX_FLAGS="-Werror -Wno-unused-command-line-argument" \
```

Comment:
As discussed, please revert this change. We want the deprecated warnings, so we know to fix the code, which is done in #172.